src/main/kotlin/org/jitsi/nlj/transform/node/PacketLossNode.kt

```diff
@@ -15,16 +15,71 @@
  */
 package org.jitsi.nlj.transform.node
 
+import org.jitsi.config.JitsiConfig
+import org.jitsi.metaconfig.config
 import org.jitsi.nlj.PacketInfo
+import org.jitsi.nlj.stats.NodeStatsBlock
+import org.jitsi.nlj.util.addRatio
 import kotlin.random.Random
 
 /**
  * A [Node] which drops packets randomly with a certain probability and a uniform distribution.
  */
-class PacketLossNode(val p: Double) : FilterNode("PacketLossNode(p=$p)") {
+class PacketLossNode(val config: PacketLossConfig) : FilterNode("PacketLossNode($config)") {
     private val random = Random(System.currentTimeMillis())
+    private var inBurst = false
+    private var currentBurstPacketsDropped = 0
 
-    override fun accept(packetInfo: PacketInfo) = random.nextDouble() >= p
+    override fun accept(packetInfo: PacketInfo) = acceptBurst() && random.nextDouble() >= config.uniformRate
```

Comment:
I think this would be clearer as:
```
override fun accept(packetInfo: PacketInfo) {
    return if (bustEnabled) {
        acceptBurst()
    } else {
        random.nextDouble() > config.uniformRate
    }
}
```
or even moving the uniform logic into an `acceptUniform` method, even though it's simple:

```
override fun accept(packetInfo: PacketInfo) = if (bustEnabled) acceptBurst() else acceptUniform()
```

EDIT: I just realized that you were probably intending for uniform to piggy-back on burst, is that what we want?

Reply:
The way I have it the uniform loss is applied after the burst (regardless of whether burst is enabled or not). I don't really know how this will be used in practice, so I'd rather keep the options of enabling both independently open.

Reply:
Ok, if we don't think the burst loss is useful we can get rid of it, I think I had a use for it back when I wrote the other PacketLoss node.  If we do keep it, It may be worth giving some thought to some uses cases to decide if we want them to stack or be exclusive.  @JonathanLennox may have some thoughts.

Reply:
I'm not sure this is the best model of burst loss (I could probably dig up some papers on it if we cared), but I could imagine it being useful, and this is good enough for now.